### PR TITLE
feat(presets): add extractVersion to regex managers

### PR DESCRIPTION
**Pull request by maxbrunet**
## Changes

As in title

## Context

I often find the need to trim a prefix from some versions.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Comment by secustor** [comment]
Needs prettier fix

File: lib/config/presets/internal/regex-managers.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const presets: Record<string, Preset> = {
         customType: 'regex',
         fileMatch: ['.+\\.tfvars$'],
         matchStrings: [
-          '#\\s*renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?\\s.*?_version\\s*=\\s*"(?<currentValue>.*)"',
+          '#\\s*renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)( versioning=(?<versioning>.*?))?(?: extractVersion=(?<extractVersion>.*?))?\\s.*?_version\\s*=\\s*"(?<currentValue>.*)"',
```
Comment (secustor) [inline lib/config/presets/internal/regex-managers.ts:81]: We do you have here a different char selector than in the other presets?

Comment (maxbrunet) [reply lib/config/presets/internal/regex-managers.ts:81]: It's consistent wihtin itself. Would you like to make this entire regex consistent with the others?

Comment (secustor) [reply lib/config/presets/internal/regex-managers.ts:81]: I see. Keep it that way in that case.

**Review by secustor — APPROVED** [review]
LGTM 

@rarkins do you see any problem with exposing this from an UX perspective?

**Review by viceice — APPROVED** [review]

**Comment by rarkins** [comment]
@secustor at first I didn't like the sounds of it, but if it's as elegant as this, I'm happy to merge it unless/until we find a problem

**Pull request merged**